fzf.el

```diff
@@ -86,11 +101,11 @@
     (split-window-vertically window-height)
     (when fzf/position-bottom (other-window 1))
     (if fzf/args
-        (apply 'make-term "fzf" fzf/executable nil (split-string fzf/args " "))
+        (apply 'make-term "fzf" fzf/executable nil fzf/args)
       (make-term "fzf" fzf/executable))
     (switch-to-buffer buf)
     (linum-mode 0)
-    (set-window-margins nil 1)
+    ;;(set-window-margins nil 1)
```

Comment:
I have been using like this for some days now and it's working fine. But I am only using emacs from terminal, so I am not aware of the implications on the x gui version. Please notice that I was working on some another features as well on this branch, maybe you will want one "clean" branch with only this change.